tests/doc_scan/request/check/sandbox.third.party.identity.check.builder.spec.js

```diff
@@ -0,0 +1,83 @@
+const SandboxThirdPartyIdentityCheck = require('../../../../src/doc_scan/request/check/sandbox.third.party.identity.check');
+
+const {
+  SandboxThirdPartyIdentityCheckBuilder,
+  SandboxRecommendationBuilder,
+  SandboxBreakdownBuilder,
+} = require('../../../..');
+
+const SOME_RECOMMENDATION = new SandboxRecommendationBuilder()
+  .withValue('some-value')
+  .build();
+
+const SOME_BREAKDOWN = new SandboxBreakdownBuilder()
+  .withResult('some-result')
+  .withSubCheck('some-check')
+  .build();
+
+describe('SandboxThirdPartyIdentityCheckBuilder', () => {
+  describe('#withRecommendation', () => {
+    it('Should build SandboxThirdPartyIdentityCheck with recommendation', () => {
+      const check = new SandboxThirdPartyIdentityCheckBuilder()
+        .withRecommendation(SOME_RECOMMENDATION)
+        .build();
+
+      expect(check)
+        .toBeInstanceOf(SandboxThirdPartyIdentityCheck);
+
+      expect(JSON.stringify(check))
+        .toEqual(JSON.stringify({
+          result: {
+            report: {
+              recommendation: SOME_RECOMMENDATION,
+              breakdown: [],
+            },
+          },
+        }));
+    });
+  });
+
+  describe('#withBreakdown', () => {
+    it('Should build SandboxThirdPartyIdentityCheck with breakdown', () => {
+      const check = new SandboxThirdPartyIdentityCheckBuilder()
+        .withRecommendation(SOME_RECOMMENDATION)
+        .withBreakdown(SOME_BREAKDOWN)
+        .build();
+
+      expect(check)
+        .toBeInstanceOf(SandboxThirdPartyIdentityCheck);
+
+      expect(JSON.stringify(check))
+        .toEqual(JSON.stringify({
+          result: {
+            report: {
+              recommendation: SOME_RECOMMENDATION,
+              breakdown: [SOME_BREAKDOWN],
+            },
+          },
+        }));
+    });
+  });
+
+  describe('#withBreakdowns', () => {
+    it('Should build SandboxThirdPartyIdentityCheck with breakdown', () => {
+      const check = new SandboxThirdPartyIdentityCheckBuilder()
+        .withRecommendation(SOME_RECOMMENDATION)
+        .withBreakdowns([SOME_BREAKDOWN])
```

Comment:
Should this bet set up to use at least 2 breakdowns ?

Reply:
Can do. But, not sure if it is needed though.

Reply:
Well, not that this is needed, but since the test just before is verifying `#withBreakdown` and this one test `#withBreakdowns` but with the same test sample and expectation than the previous one, it would look a bit more self explanatory IMO.

Reply:
Hm yes, but this one provides an array as `.withBreakdowns([SOME_BREAKDOWN])` and the previous one is just ` .withBreakdown(SOME_BREAKDOWN)`.
Thought providing an array is sufficient for the test. But yeah, we could provide one more breakdown to the array for it to be more readable.